pattern_import_export/models/ir_exports_line.py

```diff
@@ -142,9 +142,9 @@ def _check_required_fields(self):
     @api.depends("name")
     def _compute_related_level_field(self):
         for export_line in self:
-            if export_line.pattern_config_id.model_id.model and export_line.name:
+            if export_line.name:
```

Comment:
additional condition of export_line.pattern_config_id.model_id.model is needed. Without it we get a crash.